Source/ui_unix/openglwindow.cpp

```diff
@@ -0,0 +1,34 @@
+#include "openglwindow.h"
+#include <QResizeEvent>
+
+OpenGLWindow::OpenGLWindow(QWindow *parent)
+    : QWindow(parent)
+{
+    QSurfaceFormat format;
+    format.setVersion(3, 2);
+    format.setProfile(QSurfaceFormat::CompatibilityProfile);
```

Comment:
Ok, apparently, GLEW doesn't work well with core profiles. Details here: https://www.opengl.org/wiki/OpenGL_Loading_Library#GLEW_.28OpenGL_Extension_Wrangler.29. I need to do some further testing on Windows and OSX to see what would be the best way to handle this.